android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/components/RoommateFilter.kt

```diff
@@ -0,0 +1,82 @@
+package org.uwaterloo.subletr.pages.home.list.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.enums.Gender
+
+@Composable
+fun RoommateFilter(
+	currentGenderPref: Gender,
+	updateGenderFilter: (Gender) -> Unit,
+	closeAction: () -> Unit,
+) {
+	var genderPref by remember {
+		mutableStateOf(
+			currentGenderPref
+		)
+	}
+	BasicFilterLayout(
+		modifier = Modifier.height(200.dp),
```

Comment:
Do you need this or can we make it wrap content height?

Reply:
Wrap content height will make the bottom sheet to take up the entire screen.